sdk/storage/README.md

```diff
@@ -0,0 +1,116 @@
+- **Source Code Contributions:** Please follow the [contribution guidelines for Azure open source](https://azure.github.io/azure-sdk/cpp_introduction.html) for instructions about contributing to the source project.
+
+For general suggestions about Azure, use our [Azure feedback forum](http://feedback.azure.com/forums/34192--general-feedback).
+
+## Download & Install
+
+### Install Dependencies
+
+#### Windows
+
+On Windows, dependencies are managed by [Vcpkg](https://github.com/microsoft/vcpkg). You can reference the [Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) to quickly set yourself up.
+After Vcpkg is initialized and bootstrapped, you can install the dependencies:
+```BatchFile
+vcpkg.exe install libxml2:x64-windows curl:x64-windows
+```
+
+#### Unix Platforms
+
+You can use the package manager on different Unix platforms to install the dependencies. The dependencies to be installed are:
+
+  - CMake 3.15.0 and greater.
+  - libxml2.
+  - OpenSSL.
+  - libcurl.
+
+### Build from Source
+
+First, download the repository to your local folder:
+```BatchFile
+git clone https://github.com/Azure/azure-sdk-for-cpp.git
+```
+
+#### Windows
+
+##### Use CMake to generate the solution file
+
+In a new folder you created under the root directory:
+```BatchFile
+cmake .. -A x64 -DCMAKE_TOOLCHAIN_FILE=<YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake -DBUILD_CURL_TRANSPORT=ON
+cmake --build .
+```
+
+The built library will be in `<ProjectDir>\<Configuration>\` respectively for Azure Core and Azure Storage. e.g. `azure_core.lib` will be in `sdk\core\azure-core\Debug` for debug configuration.
+
+##### Use Visual Studio's Open by folder feature
+Open the root folder of the library with Visual Studio's Open folder feature.
+
+If Vcpkg is not globally integrated, then you need to open CMakeSettings.json and change the `Make toolchain file to be <YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake` and save.
+Then you can build Azure Storage libraries by selecting the target in Visual Studio, or simply build all.
+The libraries will be in `<ProjectRoot>\out\build\<Configuration>\sdk\<LibraryName>` respectively.
+
+#### Unix Platforms
+
+You can run the following command in a new folder created under the downloaded code's root folder to build the code.
+
+```BatchFile
+cmake .. -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DBUILD_STORAGE_SAMPLES=ON
+cmake --build .
+```
+Then you can consume the built library with the header files.
+make/ninja install is work in progress.
+
+
+### Via NuGet
+WIP
+TODO when ready.
+
+### Via Vcpkg
+WIP
+TODO when ready.
+
+## Dependencies
+
+### [Azure Core SDK](https://github.com/Azure/azure-sdk-for-cpp/blob/master/README.md)
+
+### [nlohmann/json](https://github.com/nlohmann/json)
+
+### [Libxml2](http://xmlsoft.org/)
```

Comment:
I meant it seems not necessary to make these 3 headings. You may want to try and compare, you can pick whichever you think it's prettier.